numpy/f2py/tests/test_abstract_interface.py

```diff
@@ -1,5 +1,5 @@
 import pytest
-from . import util
+from numpy.f2py.tests import util
```

Comment:
What is the reason for these changes?

Reply:
If you look at the CI failure in the commit before that one, it fails on one platform to import util from the fortran tests. I'm not sure why it only fails on the one platform.

Reply:
It doesn't fail with the current spin, right? So what changed with spin? Can you debug this locally? It seems like an unnecessary change.

Reply:
It wasn't an unnecessary change, per se:

https://github.com/scientific-python/spin/issues/209

`pytest` test discovery is quite complicated, unfortunately, and we have multiple ways in which people organize source (`src` vs `numpy`) and specify which they wish to run. I'll take another look next week, but the fact that none of the other projects ran into issues is what made me attempt a workaround in NumPy.

Reply:
I would prefer we have a local override for the `spin test` command that will work around the shortcomings of the global `spin test` pytest invocation.